hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobExecutionService.java

```diff
@@ -366,4 +381,21 @@ public void provideDynamicMetrics(MetricDescriptor descriptor, MetricsCollection
             ctx.provideDynamicMetrics(descriptor.copy(), context);
         });
     }
+
+    private static class Stats {
+
+        @Probe(name = MetricNames.JOB_EXECUTIONS_STARTED)
+        private final Counter executionStarted = MwCounter.newMwCounter();
+        @Probe(name = MetricNames.JOB_EXECUTIONS_TERMINATED)
+        private final Counter executionTerminated = MwCounter.newMwCounter();
+
+        public void executionStarted() {
```

Comment:
it's a little unnecessary encapsulation here, you can just increment the counters directly.